mempool/mempool.go

```diff
@@ -29,6 +29,11 @@ type Mempool interface {
 	// transactions (~ all available transactions).
 	ReapMaxTxs(max int) types.Txs
 
+	// GetNextTransaction will return transaction with condition that Bytes and Gas
+	// must be less than remainBytes and remainGas, and with highest priority less
+	// than the priority of stater
+	GetNextTransaction(remainBytes int64, remainGas int64, starter []byte) types.Tx
```

Comment:
should return `(types.Tx, error)`?

Reply:
Do we need error here? Cuz if not tx found it will return nil tx

Reply:
what if we have other types of errors, like invalid parameter or internal system error? personally I think go's error retuning style is useful in most cases, thus we should follow in most cases.